Packs/BreachRx/Integrations/BreachRx/BreachRx.yml

```diff
@@ -1,4 +1,6 @@
 category: Case Management
+sectionOrder:
+- Connect
 commonfields:
```

Comment:
Need to add collect section header too, please see [docs](https://xsoar.pan.dev/docs/integrations/yaml-file#configurations-sections)